specification/synapse/data-plane/Microsoft.Synapse/preview/2022-11-01-preview/linkConnections.json

```diff
@@ -0,0 +1,1043 @@
+          {
+            "$ref": "#/parameters/LinkConnectionName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "queryTableStatusRequest",
+            "description": "Query table status request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/QueryTableStatusRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LinkConnectionQueryTableStatus"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "../../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LinkConnection_QueryTableStatus": {
+            "$ref": "./examples/LinkConnection_QueryTableStatus.json"
+          }
+        }
+      }
+    },
+    "/linkconnections/{linkConnectionName}/updateLandingZoneCredential": {
+      "post": {
+        "tags": [
+          "linkConnections"
+        ],
+        "operationId": "LinkConnection_UpdateLandingZoneCredential",
+        "description": "Update landing zone credential of a link connection",
+        "parameters": [
+          {
+            "$ref": "#/parameters/LinkConnectionName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "updateLandingZoneCredentialRequest",
+            "description": "update landing zone credential request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/UpdateLandingZoneCredential"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "../../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LinkConnection_UpdateLandingZoneCredential": {
+            "$ref": "./examples/LinkConnection_UpdateLandingZoneCredential.json"
+          }
+        }
+      }
+    },
+    "/linkconnections/{linkConnectionName}/pause": {
+      "post": {
+        "tags": [
+          "linkConnections"
+        ],
+        "operationId": "LinkConnection_Pause",
+        "description": "Pause a link connection",
+        "parameters": [
+          {
+            "$ref": "#/parameters/LinkConnectionName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
why it is not 204?

Reply:
We want to keep consistent to existing APIs to not bring breaking changes